src/Scorecard.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+
+class Scorecard {
+
+  constructor() {
```

Comment:
Clear variable and function naming, this is so helpful when trying to read/understand the code